wren-ui/src/components/pages/modeling/metadata/ModelMetadata.tsx

```diff
@@ -52,12 +54,23 @@ export default function ModelMetadata(props: Props) {
         <FieldTable dataSource={fields} showExpandable />
       </div>
 
-      <div className="mb-6">
-        <Typography.Text className="d-block gray-7 mb-2">
-          Relationships ({relationFields.length})
-        </Typography.Text>
-        <RelationTable dataSource={relationFields} showExpandable />
-      </div>
+      {!!fields.length && (
+        <div className="mb-6">
+          <Typography.Text className="d-block gray-7 mb-2">
+            Calculated fields ({fields.length})
+          </Typography.Text>
+          <CalculatedFieldTable dataSource={fields} showExpandable />
+        </div>
+      )}
```

Comment:
I guess use `fields.length` just want to check and it will be changed to `calculatedFields.length` in another PR, right? 🤔

Reply:
Yeah, It needs to adjust & check the code block component changed by other PR when resolving conflict.
I accidentally added it to the commit.